gnd/src/main/java/com/google/android/gnd/persistence/remote/firestore/AuditInfoDoc.java

```diff
@@ -59,20 +62,20 @@
   public static AuditInfo toObject(@Nullable AuditInfoDoc doc) {
     if (doc == null || doc.clientTimeMillis == null) {
       return AuditInfo.builder()
-          .setUser(UserDoc.UNKNOWN_USER)
+          .setUser(UserMapConverter.UNKNOWN_USER)
           .setClientTimeMillis(new Date(0))
           .build();
     }
     return AuditInfo.builder()
-        .setUser(UserDoc.toObject(doc.user))
+        .setUser(UserMapConverter.toUser(Data.fromMap(doc.user)))
         .setClientTimeMillis(doc.clientTimeMillis.toDate())
         .setServerTimeMillis(Optional.ofNullable(doc.serverTimeMillis).map(Timestamp::toDate))
         .build();
   }
 
   public static AuditInfoDoc fromMutationAndUser(Mutation mutation, User user) {
     AuditInfoDoc auditInfo = new AuditInfoDoc();
-    auditInfo.user = UserDoc.fromObject(user);
+    auditInfo.user = UserMapConverter.fromUser(user).toMap();
```

Comment:
Note that nesting will become simplified once we migrate the rest of the model to use `Data`.